bcycle-backend/Program.cs

```diff
@@ -14,11 +16,13 @@ public class Program
     {
         public static void Main(string[] args)
         {
+            var date = DateTime.Now;
```

Comment:
1. Unused variable?
2. Could we move the config to `Startup.cs`?

Reply:
Done